packages/scss/src/components/_button.scss

```diff
@@ -227,40 +213,43 @@
 			top: 0;
 			vertical-align: middle;
 		}
+	}
+}
 
-		&.mod-smaller {
-			&::after {
-				font-size: 1rem;
-			}
-		}
-
-		&.mod-small {
-			&::after {
-				font-size: 1.2rem;
-			}
+// sizing for states
+.button.is-loading {
```

Comment:
Shouldn't state sizing be including in the sizing loop ? (because it is the sizing of the button that changes how the state is displayed, not the other way around, even though it doesn't change anything in terms of css)